tcpslice.c

```diff
@@ -891,13 +890,11 @@ extract_slice(struct state *states, const int numfiles, const char *write_file_n
 	last_state = 0;
 	last_hdr.ts.tv_sec = last_hdr.ts.tv_usec = 0;
 	last_hdr.caplen = last_hdr.len = 0;
-	last_pkt = (u_char*) malloc(snaplen);
+	last_pkt = (u_char*) calloc(snaplen, sizeof(u_char));
```

Comment:
`calloc(1, ...` would be more consistent with the code style.

Reply:
We are doing the multiplication anyway in calloc. It's best to use calloc as intended.